ProcessMaker/Models/ProcessRequest.php

```diff
@@ -1057,9 +1057,9 @@ public function getProcessVersionAlternativeAttribute(): string | null
     /**
      * Retrieves the destination of the first closed end event.
      *
-     * @return ?string Returns a string value representing the element destination of the first closed end event.
+     * @return ?array Returns a string value representing the element destination of the first closed end event.
```

Comment:
Fix the doc description `string` -> `array`
Does this method was introduced in this feature?
Make sure to change every where this method is used to use array instead of string.

Reply:
Hello @caleeli yes this method was introduced in this feature.